experimental/yarpl/src/yarpl/flowable/sources/Flowable_FromObservable.h

```diff
@@ -122,22 +104,23 @@ class FlowableFromObservableSubscription
    */
   class TheObserver : public yarpl::observable::Observer<T> {
```

Comment:
Maybe this is a dumb question, but why do we use the pattern where there is this internal object that proxies it's methods back to the owner? In this instance there doesn't seem to be an overlap between Observer and Subscription so the FlowableFromObservableSubscription could be both.

Is it to avoid multiple inheritance, or something to do with object lifetimes etc?